desktop/nodeManager.js

```diff
@@ -30,6 +43,24 @@ class NodeManager {
   };
 
   static hardRefresh = ({ browserWindow }) => browserWindow.reload();
+
+  static killNodeProcess = async ({ event }) => {
+    try {
+      const isDevMode = process.env.NODE_ENV === 'development' || process.env.DEBUG_PROD === 'true';
```

Comment:
DEBUG_PROD means that the app is packed as production app so it won't have miniMesh.sh start point